### PR TITLE
Small player fixes

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

**Comment by cboesch-gpsw** [comment]
Oups, this depends on https://github.com/gopro/gopro-lib-node.gl/pull/80

File: ngl-tools/player.c [hunk 1]
```diff
@@ -626,6 +626,7 @@ void player_main_loop(void)
             case SDL_USEREVENT:
                 run = handle_map[event.user.code](event.user.data1) == 0;
                 free(event.user.data1);
+                p->text_last_frame_index = -1;
```
Comment (mbouron-gpsw) [inline ngl-tools/player.c:629]: Shouldn't p->text_last_frame_index also be initialized to -1 in player_init() ?

Comment (cboesch-gpsw) [reply ngl-tools/player.c:630]: Yep that's correct, and it's already true on `master`, so I added that in a separated commit.

Comment (mbouron-gpsw) [reply ngl-tools/player.c:630]: Thanks.

**Review by mbouron-gpsw — APPROVED** [review]

**Pull request merged**